site/blog_collections/_running/2025-12-31-2025_running_year_in_review.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "2025 Running Year in Review"
+description: "A summary of my running in 2025"
+date: 2025-12-31 17:00:00 +0100
+comments: true
+tags: running ultrarunning utmb review
+draft: true
+---
+
+This is a summary of how 2025 turned out to be for me in terms of running. I have kept it very short when it comes to mentioning the races etc., otherwise it would be a neverending story 😉.
+
+# Training Volume
+According to Strava, in 2025 I ran `4850K` over `464h20m` with a total elevation gain landing on `36.5K`. That's equivalent to an average of `93K` / `8.9h` per week.
+
+{% capture table_content %}
+|------------|------------|
+| <img src="/img_running/2025-12-31/strava1.jpg" data-src="/img_running/2025-12-31/strava1.jpg" alt="" class="spotlight w-100 pl-2 pr-2" style="max-width: 350px" /> | <img src="/img_running/2025-12-31/strava2.jpg" data-src="/img_running/2025-12-31/strava2.jpg" alt="" class="spotlight w-100 pl-2 pr-2" style="max-width: 350px" /> |
+{% endcapture %}
+{{ table_content | markdownify }}
+
+The volume is slightly less than 2024 (5000K / 501h) - primarily due to Tuesday cross training in the first half of the year (stair stepper, prep for Mozart).
+
+# Achievements
+
+## Half Marathon Attempts
+This year I chased the sub-90 minute half marathon goal across three races. The results were bittersweet - while my Strava times consistently showed sub-90 performances, the official chip times were all just above the mark. Close, but not quite there! Here are the three attempts:
+
+- [Race: Cross Island Bornholm](/blog/running/2025/04/19/race_cross_island_bornholm) - Official: 1h30m10s, Strava half: 1h29m39s 😆
+- [Race: Odense City Half](/blog/running/2025/05/21/race_odense_city_half) - Official: 1h31m, Strava half: 1h29m46s 😭
+- [Race: Bestseller Aarhus City Half](/blog/running/2025/06/27/race_bestseller_aarhus_city_half) - Official: 1h30m37s, Strava half: 1h29m50s
+
+## ITRA Runner Index
+During the year I reached ITRA Advanced 4 level - a milestone that reflects the accumulation of ultra distance races and vertical gain. The journey to this level has taken me through some fantastic trails and mountains across Europe.
```

Comment:
You can change "across Europe" to just mentioning adventures in Austria (Mozart 100), Sweden (österlen and kullamannen) and trail races in Denmark . Do not take literal ... verify names etc